plugin/log/setup_test.go

```diff
@@ -19,41 +19,49 @@ func TestLogParse(t *testing.T) {
 			NameScope: ".",
 			Format:    DefaultLogFormat,
 			Class:     map[response.Class]bool{response.All: true},
+			TimeUnits: "",
 		}}},
-		{`log example.org`, false, []Rule{{
+		{`log example.org `, false, []Rule{{
```

Comment:
Not sure I understand it, but why was a space added here? And why do we expect it to have TimeUnits of `"ms"`?

Reply:
sloppy checking on my part. Will fix